subsys/testsuite/Kconfig

```diff
@@ -88,6 +88,12 @@ config COVERAGE_DUMP
 	help
 	  Dump collected coverage information to console on exit.
 
+config COVERAGE_SEMIHOST
```

Comment:
This should be a choice between `COVERAGE_DUMP` and `COVERAGE_SEMIHOST` (structure added in #96126)